React/Base/RCTDefines.h

```diff
@@ -74,6 +74,23 @@
 #define RCT_PROFILE RCT_DEV
 #endif
 
+/**
+ * Add the default Metro packager port number
+ */
+#ifndef RCT_METRO_PORT
+#define RCT_METRO_PORT 8081
+#else
+// test if RCT_METRO_PORT is empty
```

Comment:
Oh my, it this a standard way to handle this kind of problem?

Reply:
@shergin good question :) . This is the only way I could find to cleanly handle the empty string dilemma.  I thought at first that we could use a bash expansion trick where you say `${RCT_METRO_PORT:=8081}` but xcode did not support the default value there.  I tried about 10 other ways that I found on varies stack overflow/exchange posts but this was by far the cleanest.  I am open to suggestions, but this works.